app/models/paper.rb

```diff
@@ -10,6 +10,12 @@ class Paper < ApplicationRecord
   validates_associated :authors
   validate :allowed_biases, :allowed_methodologies
 
+  before_save :downcase_name
+
+  def downcase_name
+    self.publication.downcase!
+  end
```

Comment:
I think I'll leave it as is in case we decide we need to downcase anything else in the future for paper like titles, tags, or authors